src/bootstrap.ts

```diff
@@ -102,6 +92,14 @@ async function connect(delayMillis: number, mongoUrl: string, username: string,
 	}
 }
 
+export async function createConnection(dbUrl: string) {
+	return mongoose.createConnection(`${dbUrl}`, {
+		directConnection: true,
+		socketTimeoutMS: 10000,
+		connectTimeoutMS: 30000,
+	});
+}
+
```

Comment:
this is only used in test utils.

If the requirement is to test the db connection functionality, then it would make sense having it in `src` and exporting.

There are already connection functions here, so I think the intended use is for the rewritten test utils. 
Please let me know if that's incorrect.